packages/esm-patient-chart-app/src/config-schema.ts

```diff
@@ -42,6 +42,19 @@ export const esmPatientChartSchema = {
     _description: 'Custom label to use on the start visit actions',
     _default: '',
   },
+  visitAttributeTypes: {
+    _type: Type.Array,
+    _description: 'List of visit attribute types to be shown when filling visit form',
+    _default: [],
+    _elements: {
+      uuid: {
+        _type: Type.UUID,
+      },
+      required: {
+        _type: Type.Boolean,
+      },
+    },
+  },
   showServiceQueueFields: {
```

Comment:
I'd argue for the need for a configuration property that determines whether to show these visit attribute types or not. Given that this visit form gets used by all implementations, not everyone will want to have to display these additional fields when starting a visit. We'd then check for this configuration property before rendering the `VisitAttributeTypeFields` component in a section in the form.  Going with the approach taken in https://github.com/openmrs/openmrs-esm-patient-chart/pull/870, that property could be called something like `showVisitAttributeTypeFields`.

Reply:
Sure @denniskigen!
I will include another config for this behaviour.
Thanks!